ibm_catalog.json

```diff
@@ -154,6 +154,216 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "deploy-arch-ibm-cos",
+            "label": "Cloud Object Storage",
+            "product_kind": "solution",
+            "tags": [
+                "dev_ops",
+                "target_terraform",
+                "terraform",
+                "solution",
+                "storage"
+            ],
+            "keywords": [
+                "terraform",
+                "cos",
+                "bucket",
+                "storage",
+                "solution"
+            ],
+            "short_description": "Creates and configures an IBM Cloud Object Storage instance and bucket",
+            "long_description": "This architecture supports creating an IBM Cloud Object Storage instance and bucket. You can also configure the following aspects of your instances: CBR rules, KMS encryption, Activity tracking and Monitoring",
+            "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/main/README.md",
+            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/images/cos_icon.svg",
+            "provider_name": "IBM",
+            "features": [
+                {
+                    "title": "Create an IBM Cloud Object Storage instance",
+                    "description": "Create an IBM Cloud Object Storage instance."
+                },
+                {
+                    "title": "Create an IBM Cloud Object Storage bucket",
+                    "description": "Create either a cross-regional or regional Object Storage bucket."
+                },
+                {
+                    "title": "Secure access with CBR",
+                    "description": "Create context-based restriction (CBR) rules for both instances and buckets."
+                },
+                {
+                    "title": "Attach access tags",
+                    "description": "Attach access tags to the IBM Cloud Object Storage instance."
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Instance",
+                    "name": "instance",
+                    "working_directory": "solutions/instance",
```

Comment:
Not sure, if we need to add here -
`install_type` as `fullstack` 
Please check.